### PR TITLE
Add script to repair invalid gc manifests

### PR DESCRIPTION
**Pull request by shino**
This PR addresses the remaining part of https://github.com/basho/riak_cs/issues/827 .
( #964 is another part)

This PR adds a script to repair invalid state manifests in GC bucket when
- manifest in GC bucket is `active` and
- corresponding actual manifest (in `0o:*` bucket) is `pending_delete`, `scheduled_delete` or `notfound`

### CODE REVIEW

**Comment by shino** [comment]
Still WIP, now debugging...
Active state remain after single run of repair script :crying_cat_face: 
https://gist.github.com/shino/8940d7bacfa2b2fbb9e7

**Comment by shino** [comment]
I hope this is ready for review.
The bug I described above can not be reproduced in riak_test.

**Comment by shino** [comment]
Fix riak_test `gc_tests` failure in multibag config by 767fbc0.

Please review with https://github.com/basho/riak_cs_multibag/pull/13 (for `{flavor, {multibag, disjoint}}` )

**Comment by kuenishi** [comment]
If we use this script in multibag, do we have to run this script in the CS node directly connecting to the Riak cluster where invalid state manifests are in GC bucket?

File: priv/tools/repair_gc_bucket.erl [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+        undefined ->
+            getopt:usage(option_spec(), "riak-cs escript /path/to/repair_gc_bucket.erl"),
+            halt(1);
+        Host ->
+            Port = proplists:get_value(port, Options),
+            debug("Connecting to Riak ~s:~B...", [Host, Port]),
+            case riakc_pb_socket:start_link(Host, Port) of
+                {ok, Pbc} ->
+                    repair(Pbc, Options),
+                    timer:sleep(100);
```
Comment (kuenishi) [inline priv/tools/repair_gc_bucket.erl:79]: I prefer `riakc_pb_socket:stop/1` called here, not in the tail of function call chain.

**Comment by kuenishi** [comment]
Ah, I got it. It doesn't access riak via CS, but directly hits riak.

File: rebar.config [hunk 2]
```diff
@@ -54,5 +54,5 @@
 
 {deps_ee, [
            {riak_repl_pb_api,".*",{git,"git@github.com:basho/riak_repl_pb_api.git", {tag, "0.2.5"}}},
-           {riak_cs_multibag,".*",{git,"git@github.com:basho/riak_cs_multibag.git", {tag, "1.5.0"}}}
+           {riak_cs_multibag,".*",{git,"git@github.com:basho/riak_cs_multibag.git", {branch, "master"}}}
```
Comment (kuenishi) [inline rebar.config:57]: Don't forget to update this before code freeze!

**Comment by kuenishi** [comment]
@borshop merge

**Pull request merged**